### PR TITLE
fix: og tags of portfolio

### LINKED ISSUE #399: fix: og tags of portfolio
the og tag of portfolio is completely broken.

Make sure while sharing the article it must show

-> user name
-> user profile pic

additional fixes
-> look into the logic of calculating year of the content team member.

### PR DESCRIPTION
**Pull request by anish-patro**
### Your checklist for this pull request

🚨Please review the [guidelines for contributing](https://github.com/Monday-Morning/project-tahiti/blob/main/CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your default branch!
- [x] Make sure you are making a pull request against the **default branch** (left side). Also you should start _your branch_ off _default branch_.
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.
- [ ] I have added necessary documentation (if appropriate)

### Linked Issues

Fixes #399 

### Description

Please describe your pull request.

❤️Thank you!

### Post merge checklist

- [ ] Follow steps from the [guidelines for contributing](https://github.com/Monday-Morning/project-tahiti/blob/main/CONTRIBUTING.md) to this repository.
- [ ] If you are a new contributor, ping in the thread and one of the maintainers will add you to all-contributors list.

### CODE REVIEW

File: client/src/pages/portfolio/[...userId].jsx [hunk 1]
```diff
@@ -137,7 +137,7 @@ const PortfolioPage = ({
         />
         <meta
           property='og:title'
-          content={`{firstName + ' ' + lastName} | Monday Morning`}
+          content={`${firstName + ' ' + lastName} | Monday Morning`}
```
Comment (120EE0692) [inline client/src/pages/portfolio/[...userId].jsx:140]: @anish-patro add a condition check lastName should not be null or undefined.

**Review by 120EE0692 — CHANGES_REQUESTED** [review]
@anish-patro add the same check in the title as well.

**Comment by sambit-sankalp** [comment]
@anish-patro updates on the changes requested??

**Review by 120EE0692 — APPROVED** [review]

**Review by rutajdash — APPROVED** [review]

**Comment by rutajdash** [comment]
@anish-patro please update the branch with latest commits

**Review by sambit-sankalp — APPROVED** [review]

**Pull request merged**